docs/source/models/gbd2017/causes/lower_respiratory_infections/index.rst

```diff
@@ -46,10 +49,104 @@ Data Description
 
    * - State
      - Definition
+   * - S
+     - Susceptible but does not currently have LRI
    * - I
      - Currently infected and having the condition
+
+.. list-table:: **States Data**
+   :widths: 20 25 30 30
+   :header-rows: 1
+
+   * - State
+     - Measure
+     - Value
+     - Notes
    * - S
-     - Susceptible but does not currently have LRI
+     - prevalence
+     - 1-prevalence_c322
+     -
+   * - S
+     - excess mortality |br| rate
+     - 0
+     -
+   * - S
+     - disabilty weights
+     - 0
+     -
+   * - I
+     - prevalence
+     - prevalence_c322
+     -
+   * - I
+     - excess mortality |br| rate
+     - :math:`\frac{\text{deaths_c322}}{\text{population * prevalence_c322}}`
+     -
+   * - I
+     - disability weights
+     - disability_weight_s670* |br| prevalence_s670+ |br| disability_weight_s669* |br| prevalence_s669
+     - GBD assumes 15% of LRI |br| cases as severe and 85% |br| as moderate |br| [GBD-2017-YLD-Capstone-Appendix-1]_.
+   * - ALL
+     - cause specific |br| mortality rate
+     - :math:`\frac{\text{deaths_c322}}{\text{population}}`
+     -
+
+.. list-table:: **Transition Data**
+   :widths: 10 10 10 30 30
+   :header-rows: 1
+
+   * - Transition
+     - Source
+     - Sink
+     - Value
+     - Notes
+   * - i
+     - S
+     - I
+     - incidence_rate_c322
```

Comment:
I think we've been just labeling this as "incidence_c322", but we can revisit this, and if we're in agreement on that?

I was also writing this as incidence_c322/(1-prevalence_c322), to convert it to incidence within the susceptible population. I wasn't sure from @alibow 's documentation on incidence if the software engineers do this conversion themselves, or if we should do it?

Reply:
I think we should explicitly put in the conversion of dividing by 1-prevalence, since the numbers we can directly pull from GBD are the "total population incidence rate" and the prevalence. I think Vivarium Inputs is written to do this transformation automatically, but we need to explicitly say whether this is what we want or if we want something different.

Reply:
> I think we've been just labeling this as "incidence_c322", but we can revisit this, and if we're in agreement on that?

What do you think @beatrixkh ? I think James recommended leaving out "rate" since this is the standard assumption, but I lean towards being more explicit and calling it incidence_rate.